### PR TITLE
(feat, netfilx) Create Fast Property with mutiple scopes

### PR DESCRIPTION
**Pull request by zanthrash**
When creating a Fast Property the user can select multiple scopes and the
property will created for each scope. This will create a Persisted Property stage
for each scope.

### CODE REVIEW

File: app/scripts/modules/netflix/fastProperties/domain/propertyCommand.model.ts [hunk 1]
```diff
@@ -36,36 +37,45 @@ export class PropertyCommand {
...
+  //   return this.scope.forSubmit(this.property.env);
+  // }
 
   public originalScopeForSubmit(): Scope {
-    return this.originalScope ? this.originalScope.forSubmit(this.property.env) : this.scopeForSubmit();
+    return this.originalScope ? this.originalScope.forSubmit(this.property.env) : null;
   }
 
+  public getCombinedInstanceCountsForAllScopes(): number {
+    return this.scopes.reduce((count: number, scope: Scope) => {
```
Comment (duchuyctlk) [inline app/scripts/modules/netflix/fastProperties/domain/propertyCommand.model.ts]: Could this be rewritten as below:

```javascript
return this.scopes.reduce(((count: number, scope: Scope) => count + scope.instanceCounts.up), 0);
```
?

File: app/scripts/modules/netflix/fastProperties/scope/fastPropertyScopeSearchCategory.service.ts [hunk 2]
```diff
@@ -103,7 +103,7 @@ export class FastPropertyScopeCategoryService {
   public impactCountForRegions(scope: Scope): ng.IPromise<IImpactCounts> {
     return this.fastPropertyReader.fetchImpactCountForScope(scope)
       .then((results: any) => {
-        return <IImpactCounts>{total: results.count, up: results.count};
+        return <IImpactCounts>{total: parseInt(results.count, 10), up: parseInt(results.count, 10)};
```
Comment (icfantv) [inline app/scripts/modules/netflix/fastProperties/scope/fastPropertyScopeSearchCategory.service.ts]: I think we can disable the radix linter warning for this as base 10 is the default so if you wanna drop it here, that's cool.  You'll need to set `"radix": false` in `tslint.json`.

File: app/scripts/modules/netflix/fastProperties/wizard/propertyScope/propertyScope.component.html [hunk 3]
```diff
@@ -33,126 +36,137 @@
...
-                  ng-if="$ctrl.isEditing"
-                  type="text" ng-model="$ctrl.selectedScope.appId">
-              </td>
-            </tr>
-            <tr ng-if="$ctrl.selectedScope.region">
-              <td><b>Region</b></td>
-              <td>
-                <span ng-if="!$ctrl.isEditing">
-                  {{$ctrl.selectedScope.region}}
+                    <input
```
Comment (icfantv) [inline app/scripts/modules/netflix/fastProperties/wizard/propertyScope/propertyScope.component.html]: I think the indent is off here.

File: app/scripts/modules/netflix/fastProperties/wizard/propertyScope/propertyScope.component.html [hunk 4]
```diff
@@ -33,126 +36,137 @@
...
-                  ng-if="$ctrl.isEditing"
-                  type="text" ng-model="$ctrl.selectedScope.region">
-              </td>
-            </tr>
-            <tr ng-if="$ctrl.selectedScope.stack">
-              <td><b>Stack</b></td>
-              <td>
-                <span ng-if="!$ctrl.isEditing">
-                  {{$ctrl.selectedScope.stack}}
+                    <input
```
Comment (icfantv) [inline app/scripts/modules/netflix/fastProperties/wizard/propertyScope/propertyScope.component.html]: ditto.

File: app/scripts/modules/netflix/fastProperties/wizard/propertyScope/propertyScope.component.html [hunk 5]
```diff
@@ -33,126 +36,137 @@
...
-                  ng-if="$ctrl.isEditing"
-                  type="text" ng-model="$ctrl.selectedScope.stack">
-              </td>
-            </tr>
-            <tr ng-if="$ctrl.selectedScope.cluster">
-              <td><b>Cluster</b></td>
-              <td>
-                <span ng-if="!$ctrl.isEditing">
-                  {{$ctrl.selectedScope.cluster}}
+                    <input
```
Comment (icfantv) [inline app/scripts/modules/netflix/fastProperties/wizard/propertyScope/propertyScope.component.html]: ditto

File: app/scripts/modules/netflix/fastProperties/wizard/propertyScope/propertyScope.component.html [hunk 6]
```diff
@@ -33,126 +36,137 @@
...
+                  <td>
+                <span ng-if="!scope.isEditing">
+                  {{scope.cluster}}
                 </span>
-                <input
-                  class="form-control"
-                  ng-if="$ctrl.isEditing"
-                  type="text" ng-model="$ctrl.selectedScope.cluster">
-              </td>
+                    <input
```
Comment (icfantv) [inline app/scripts/modules/netflix/fastProperties/wizard/propertyScope/propertyScope.component.html]: here too

Comment (icfantv) [inline app/scripts/modules/netflix/fastProperties/wizard/propertyScope/propertyScope.component.html]: and a few other places further down...

**Pull request merged**